test/busted/outputHandlers/nvim.lua

```diff
@@ -241,45 +252,21 @@ return function(options)
       string = skippedString
     elseif status == 'failure' then
       failureCount = failureCount + 1
-      string = nil
+      string = failureString .. failureDescription(handler.failures[#handler.failures])
     elseif status == 'error' then
       errorCount = errorCount + 1
-      string = nil
-    end
-
-    if string ~= nil then
-      if elapsedTime_ms == elapsedTime_ms then
-        string = timeString:format(elapsedTime_ms) .. ' ' .. string
-      end
-      io.write(string)
-      io.flush()
+      string = errorString .. failureDescription(handler.errors[#handler.errors])
+    else
+      string = "unexpected test status! ("..status..")"
     end
+    write_status(element, string)
 
     return nil, true
   end
 
-  handler.testFailure = function(_element, _parent, _message, _debug)
-    io.write(failureString)
-    io.flush()
-
-    io.write(failureDescription(handler.failures[#handler.failures]))
-    io.flush()
-    return nil, true
-  end
-
-  handler.testError = function(_element, _parent, _message, _debug)
-    io.write(errorString)
-    io.flush()
-
-    io.write(failureDescription(handler.errors[#handler.errors]))
-    io.flush()
-    return nil, true
-  end
-
   handler.error = function(element, _parent, _message, _debug)
     if element.descriptor ~= 'it' then
```

Comment:
is this still wanted?

Reply:
`handler.error`, or the `element.descriptor ~= 'it'`?

It is used with errors/failures in "describe" for example, and used as a general fallback.

Test file:
```lua
local helpers = require('test.functional.helpers')(after_each)
local eq = helpers.eq

describe('it_fails', function()
  it('fails', function()
    eq(5, 6)
  end)
end)

describe('it_errors', function()
  it('errors', function()
    meh()
  end)
end)

describe('describe_error', function()
  meh()
end)

describe('describe_fail', function()
  eq(1, 2)
end)
```

Reply:
> `handler.error`, or the `element.descriptor ~= 'it'`?

The `element.descriptor ~= 'it'`

Reply:
It will still get to `handler.error` - like previously.  (IIRC)